apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -19,23 +19,35 @@ var styles = {
   }
 };
 
-var MarkdownInstructions = function (props) {
-  return (
-    <div
-      className='instructions-markdown'
-      style={[
-        styles.standard,
-        props.inTopPane && styles.inTopPane,
-        props.markdownClassicMargins && styles.classic
-      ]}
-      dangerouslySetInnerHTML={{ __html: props.renderedMarkdown }}/>
-  );
-};
+const MarkdownInstructions = React.createClass({
+  propTypes: {
+    renderedMarkdown: React.PropTypes.string.isRequired,
+    markdownClassicMargins: React.PropTypes.bool,
+    inTopPane: React.PropTypes.bool
+  },
 
-MarkdownInstructions.propTypes = {
-  renderedMarkdown: React.PropTypes.string.isRequired,
-  markdownClassicMargins: React.PropTypes.bool,
-  inTopPane: React.PropTypes.bool
-};
+  componentDidMount() {
+    // If we have the jQuery details plugin, enable it's usage on any details
+    // elements
+    const detailsDOM = $(ReactDOM.findDOMNode(this)).find('details');
+    if (detailsDOM.length && detailsDOM.details) {
+      detailsDOM.details();
```

Comment:
Since `detailsDOM` is a jQuery object, you shouldn't need to check its length here - I think just calling `details()` on an empty set would be idiomatic.